.github/workflows/release.yml

```diff
@@ -0,0 +1,78 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+name: Release
+on:
+  push:
+    tags:
+      - kani-*
+
+jobs:
+  Release:
+    name: Release
+    runs-on: ubuntu-20.04
```

Comment:
You're bundling in `ubuntu-20.04` and uploading for both `ubuntu-20.04` and `macos-10.15`. Shouldn't you be bundling in `macos-10.15` as well or am I missing something?

Reply:
There's two jobs here: Release and Upload. Release does nothing but say "hey github, create a release"

The bundling and upload of a tarball happens in the other job.